CMakeLists.txt

```diff
@@ -47,10 +42,10 @@ ament_target_dependencies(${PROJECT_NAME} PUBLIC
 )
 
 install(TARGETS ${PROJECT_NAME}
-  EXPORT ${PROJECT_NAME}
+  EXPORT ${PROJECT_NAME}Targets
```

Comment:
```suggestion
  EXPORT export_${PROJECT_NAME}
```

Reply:
I think this usage is more common

Reply:
I don't know where the name `export_${PROJECT_NAME}` came from, but multiple known libraries ([gtest](https://github.com/google/googletest/blob/master/googletest/CMakeLists.txt#L102), [fmt](https://github.com/fmtlib/fmt/blob/master/CMakeLists.txt#L315), and [nlohmann_json](https://github.com/nlohmann/json/blob/develop/CMakeLists.txt#L53)) use either `{PROJECT_NAME}Targets` or `${PROJECT_NAME}-targets` which IMO is a better name for an export set

Reply:
the export_ thing came from rclcpp, I copied it into moveit2.

Reply:
I take that back, rclcpp doesn't do that... I think it came from the ros2 documentation on ament cmake... but now I can't find it.

Reply:
https://docs.ros.org/en/foxy/Guides/Ament-CMake-Documentation.html#building-a-library

That Is where the pattern came from.  If `{PROJECT_NAME}Targets` or `${PROJECT_NAME}-targets` is more standard, please change it in moveit2 and elsewhere.